Sources/App/configure.swift

```diff
@@ -0,0 +1,23 @@
+{{#leaf}}import Leaf
+{{/leaf}}import MongoDBVapor
+import Vapor
+
+/// Configures the application.
+public func configure(_ app: Application) throws {
+    // Uncomment to serve files from the /Public folder.
+    // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory))
+    {{#leaf}}
+    // Use LeafRenderer for views.
+    app.views.use(.leaf)
+    {{/leaf}}
+    // Use `ExtendedJSONEncoder` and `ExtendedJSONDecoder` for encoding/decoding `Content`. We use extended JSON both
+    // here and on the frontend to ensure all MongoDB type information is correctly preserved.
+    // See: https://docs.mongodb.com/manual/reference/mongodb-extended-json{{#leaf}}
+    // Note that for _encoding_ content, this encoder only gets used for the REST API methods, since Leaf uses its own
+    // custom encoder to encode data for rendering in Leaf views.{{/leaf}}
+    ContentConfiguration.global.use(encoder: ExtendedJSONEncoder(), for: .json)
+    ContentConfiguration.global.use(decoder: ExtendedJSONDecoder(), for: .json)
```

Comment:
I don't think we're currently using extended JSON in the leaf frontend, and I guess it's up to the user to use it manually via curl for the non-leaf one. Maybe in the non-leaf version we don't use extended JSON and in the leaf one we should via js-bson? Not sure what's the best path here.

Reply:
We could make the server handle all the date operations kind of like we do with `_id` now, so that the user can POST stuff without having to specify an extJSON date but they'll see them when they GET the results (e.g. a `createdAt` field or something).

Reply:
done ✅